testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1147,6 +1147,11 @@ public void testCreateSchema()
         assertQueryFails("DROP SCHEMA " + schemaName, format("line 1:1: Schema '.*\\.%s' does not exist", schemaName));
     }
 
+    protected String prepareCreateSchemaStatement(String schemaName)
```

Comment:
This is not really a prepare statement (as the name may imply).
Use a rather more appropriate name:  e.g. : `createSchemaSql(String schemaName)`